src/theme.tsx

```diff
@@ -59,21 +59,23 @@ const customTheme = {
         title: "#1F2933", // gray.900
         titleHover: "#DC1853", // red.500
         text: "#3E4C59", // gray.700
+        alertText: "#5F370E", // yellow.900
         tertiary: "#616E7C", // gray.500
         icon: "#E4E7EB", // gray.100
-        link: "#264AD9", // blue.500
+        link: "#264AD9" // blue.500
       },
       dark: {
         background: "#323F4B", // gray.800
         card: "#1F2933", // gray.900
         title: theme.colors.white,
         titleHover: "#F0759A", // red.300
         text: "#CBD2D9", // gray.200
+        alertText: "#FEFCBF", // yellow.100
```

Comment:
This is totally one way to do this! Since alerts aren't super common but will likely be in other (success, info, and danger) colors, I'd offer the possibility to do this inline. So:
```
color={colorMode === 'light' ? 'yellow.900' : 'yellow.50' }
```

Reply:
Absolutely makes more sense in this case. Thanks!